backend/app/services/adapters/executor_kinds.py

```
@@ -1140,6 +1140,41 @@ def _format_subtasks_response(
+                additional_skills_json = task_crd.metadata.labels.get(
+                    "additionalSkills"
+                )
+                if additional_skills_json:
+                    try:
+                        import json as json_module
+
+                        user_selected_skills = json_module.loads(additional_skills_json)
+                        logger.info(
+                            f"[EXECUTOR_DISPATCH] task_id={subtask.task_id} user_selected_skills={user_selected_skills}"
+                        )
+                    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)